packages/components/src/components/tooltip/tooltip.tsx

```diff
@@ -30,6 +32,7 @@ const Tooltip = ({
             className={classNames(className, 'dc-tooltip', { 'dc-tooltip--error': has_error })}
             data-tooltip={message || undefined}
             data-tooltip-pos={alignment}
+            onClick={onClickMessage}
```

Comment:
shouldn't there be a default value? or call it as () => onClickMessage?.()

Reply:
ok i checked, because the type of onClick can be typed as undefined, we don't need to set it up as you had suggested, we can just pass it as is.

![Screenshot 2024-05-27 at 5 22 35 PM](https://github.com/binary-com/deriv-app/assets/103412909/cd2daf98-1068-4059-9a5d-068c3863fdd0)